language-support/ts/daml-ledger/index.test.ts

```diff
@@ -133,36 +134,24 @@ describe("streamQuery", () => {
 
   test("receive warnings", () => {
     const ledger = new Ledger(mockOptions);
-    const stream = ledger.streamQuery(Foo);
+    const stream = ledger.streamQueries(Foo, []);
     stream.on("change", mockChange);
     const restoreConsole = mockConsole();
     mockInstance.serverSend({ warnings: ["oh oh"] });
-    expect(console.warn).toHaveBeenCalledWith("Ledger.streamQuery warnings", {"warnings": ["oh oh"]});
+    expect(console.warn).toHaveBeenCalledWith("Ledger.streamQueries warnings", {"warnings": ["oh oh"]});
     restoreConsole();
   });
 
   test("receive errors", () => {
     const ledger = new Ledger(mockOptions);
-    const stream = ledger.streamQuery(Foo);
+    const stream = ledger.streamFetchByKey(Foo, fooKey);
```

Comment:
Why did you change this?

Reply:
Because this is testing the code in the underlying `streamSubmit` function, so the external function we call doesn't matter here, and I wanted to make that explicit, rather than just redo the exact same test with each of the four public methods, all targeted specifically to testing the exact same behaviour of the underlying private method.